eZ/Publish/Core/FieldType/Tests/RelationTest.php

```diff
@@ -360,6 +378,31 @@ public function testGetRelations()
         );
     }
 
+    public function testValidateNotExistingContentRelation(): void
+    {
+        $destinationContentId = 'invalid';
+
+        $this->contentServiceMock
+            ->expects(self::once())
+            ->method('loadContent')
+            ->with($destinationContentId)
+            ->willThrowException(new NotFoundException('content', $destinationContentId));
+
+        $validationErrors = $this->doValidate([], new Value($destinationContentId));
+
+        $this->assertIsArray($validationErrors);
+        $this->assertEquals([
+            new ValidationError(
+                'Content with identifier %contentId% is not a valid relation target',
```

Comment:
Don't forget to update translations if necessary.

Reply:
What translations do you have in mind?

Reply:
> What translations do you have in mind?

Repository errors translations afaics are defined here: `eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.en.xlf`

Reply:
This is pretty interesting as no validation error messages are located in this file at all. Those errors are just bunched together later on and one error is showing up which is `Content fields did not validate`. Do we really require translations for those strings as they are not all showing in the UI?

Reply:
pov ping @mikadamczyk

Reply:
As I see other error messages in such cases are not translated. Perhaps it should but: after discussion, for all of them, and in separate PR